types/emoji-js/emoji-js-tests.ts

```diff
@@ -0,0 +1,24 @@
+import EmojiConvertor from 'emoji-js';
```

Comment:
Since there's no `default`, you need to change this import to `import EmojiConvertor = require('emoji-js');`